packages/theme/css/characteristics/text.css

```diff
@@ -3,7 +3,7 @@
   --salt-text-letterSpacing: 0;
   --salt-text-textAlign: left;
   --salt-text-textAlign-embedded: center;
-  --salt-text-textDecoration: none;
+  --salt-text-textDecoration: var(--salt-typography-textDecoration-none);
```

Comment:
Should this be deprecated alongside the navigable characteristics?

Reply:
Yes, updated to deprecation